extern/algebra-plugins/CMakeLists.txt

```diff
@@ -12,11 +12,17 @@ include( FetchContent )
 message( STATUS "Building Algebra Plugins as part of the Detray project" )
 
 # Declare where to get Algebra Plugins from.
-set( DETRAY_ALGEBRA_PLUGINS_SOURCE
-   "URL;https://github.com/acts-project/algebra-plugins/archive/refs/tags/v0.8.0.tar.gz;URL_MD5;33554a09c16327078ceae1d70b525411"
-   CACHE STRING "Source for Algebra Plugins, when built as part of this project" )
-mark_as_advanced( DETRAY_ALGEBRA_PLUGINS_SOURCE )
-FetchContent_Declare( AlgebraPlugins ${DETRAY_ALGEBRA_PLUGINS_SOURCE} )
+#set( DETRAY_ALGEBRA_PLUGINS_SOURCE
+#   "URL;https://github.com/acts-project/algebra-plugins/archive/refs/tags/v0.9.2.tar.gz;URL_MD5;7965f746a28381a1bccaebd2133284bf"
+#   CACHE STRING "Source for Algebra Plugins, when built as part of this project" )
+#mark_as_advanced( DETRAY_ALGEBRA_PLUGINS_SOURCE )
+#FetchContent_Declare( AlgebraPlugins ${DETRAY_ALGEBRA_PLUGINS_SOURCE} )
+
+FetchContent_Declare( AlgebraPlugins 
+ GIT_REPOSITORY https://github.com/beomki-yeo/algebra-plugins.git
```

Comment:
Why exactly is this necessary?

Reply:
No worries. I will use regular release once acts-project/algebra-plugins#61